Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -1538,4 +1539,41 @@ theorem multiplicity_normalizedFactorsEquivSpanNormalizedFactors_symm_eq_multipl
     multiplicity_normalizedFactorsEquivSpanNormalizedFactors_eq_multiplicity hr ha]
 #align multiplicity_normalized_factors_equiv_span_normalized_factors_symm_eq_multiplicity multiplicity_normalizedFactorsEquivSpanNormalizedFactors_symm_eq_multiplicity
 
+/-- The bijection `normalizedFactorsEquivSpanNormalizedFactors.symm` between the set of prime
+    factors of the ideal `⟨r⟩` and the set of prime factors of `r` preserves `count` of the
+    corresponding multisets. See
+    `multiplicity_normalizedFactorsEquivSpanNormalizedFactors_eq_multiplicity` for the version
+    stated in terms of multiplicity. -/
+theorem count_normalizedFactorsSpan_eq_count {r X : R} (hr : r ≠ 0) (hX₁ : normUnit X = 1)
+    (hX : Prime X) : Multiset.count X (normalizedFactors r) =
+      Multiset.count (Ideal.span {X} : Ideal R) (normalizedFactors (Ideal.span {r})) := by
```

Comment:
You could make this
```suggestion
theorem count_normalizedFactorsSpan_eq_count' {r X : R} (hr : r ≠ 0) (hX : Prime X) :
    Multiset.count (normalize X) (normalizedFactors r) =
      Multiset.count (Ideal.span {X} : Ideal R) (normalizedFactors (Ideal.span {r})) := by
```
and then prove your original theorem with
```
  simpa [hX₁] using count_normalizedFactorsSpan_eq_count' hr hX
```

Reply:
Done, thanks!